packages/passport-interface/src/api/requestCheckTicketById.ts

```diff
@@ -0,0 +1,36 @@
+import urlJoin from "url-join";
+import {
+  CheckTicketByIdRequest,
+  CheckTicketByIdResponseValue,
+  TicketError
+} from "../RequestTypes";
+import { APIResult } from "./apiResult";
+import { httpPost } from "./makeRequest";
+
+/**
+ * Does NOT check in the user, rather checks whether the ticket is valid and
+ * can be used to check in.
```

Comment:
worth clarifying which ticket issuers this is applicable for - Devconnect, Zuconnect, or both?